SlicerLite/SlicerLiteLib/ItemModel.py

```diff
@@ -0,0 +1,77 @@
+import qt
+import slicer
+
+from .SlicerUtils import *
+
+
+class DICOMItem:
```

Comment:
Not sure it's a DICOM Item. Seems more like a VolumeHierarchy.

This also would apply to Data loaded which are not DICOMs